client/src/components/setup/record-idle-slide.tsx

```diff
@@ -20,7 +21,10 @@ export function RecordIdleSlide(props: {
 
   function onRecord() {
     navigate(
-      `/record?videoId=${idle.question._id}&back=${encodeURI(`/setup?i=${i}`)}`
+      urlBuild("/record", {
+        videoId: idle.question._id,
+        back: encodeURI(`/setup?i=${i}`),
```

Comment:
trying to think this through: do we need to encode this param here (and it gets encoded another time by `urlBuild` right?)?